README.md

```diff
@@ -81,6 +81,10 @@ Set `PYRIGHT_PYTHON_DEBUG` to any value.
 
 Set `PYRIGHT_PYTHON_FORCE_VERSION` to the desired version, e.g. `1.1.156`, `latest`
 
+### Keeping Pyright and Pylance in sync
+
+Set `PYRIGHT_PYTHON_PYLANCE_VERSION` to your Pylance version, e.g. `2023.11.11`, `latest-release`, `latest-prerelease`. The corresponding Pyright version will be used. Note that `PYRIGHT_PYTHON_FORCE_VERSION` takes precedence over `PYRIGHT_PYTHON_PYLANCE_VERSION`, so you'll want to set one or the other, not both.
```

Comment:
nit: would be great to link to the pylance release repo or changelogs so users can easily view all of the available versions.